catalog/env.template

```diff
@@ -26,17 +26,17 @@ AIRFLOW_VAR_ENVIRONMENT=dev
 # API Keys
 ########################################################################################
 # See: https://airflow.apache.org/docs/apache-airflow/stable/howto/variable.html#storing-variables-in-environment-variables
-AIRFLOW_VAR_API_KEY_BROOKLYN_MUSEUM=not_set
-AIRFLOW_VAR_API_KEY_DATA_GOV=not_set
-AIRFLOW_VAR_API_KEY_EUROPEANA=not_set
-AIRFLOW_VAR_API_KEY_FLICKR=not_set
-AIRFLOW_VAR_API_KEY_FREESOUND=not_set
-AIRFLOW_VAR_API_KEY_JAMENDO=not_set
-AIRFLOW_VAR_API_KEY_JUSTTAKEITFREE=not_set
-AIRFLOW_VAR_API_KEY_NYPL=not_set
-AIRFLOW_VAR_API_KEY_RAWPIXEL=not_set
-AIRFLOW_VAR_API_KEY_THINGIVERSE=not_set
-AIRFLOW_VAR_API_KEY_WALTERS_ART_MUSEUM=not_set
+#AIRFLOW_VAR_API_KEY_BROOKLYN_MUSEUM=not_set
+#AIRFLOW_VAR_API_KEY_DATA_GOV=not_set
+#AIRFLOW_VAR_API_KEY_EUROPEANA=not_set
+#AIRFLOW_VAR_API_KEY_FLICKR=not_set
+#AIRFLOW_VAR_API_KEY_FREESOUND=not_set
+#AIRFLOW_VAR_API_KEY_JAMENDO=not_set
+#AIRFLOW_VAR_API_KEY_JUSTTAKEITFREE=not_set
+#AIRFLOW_VAR_API_KEY_NYPL=not_set
+#AIRFLOW_VAR_API_KEY_RAWPIXEL=not_set
+#AIRFLOW_VAR_API_KEY_THINGIVERSE=not_set
+#AIRFLOW_VAR_API_KEY_WALTERS_ART_MUSEUM=not_set
```

Comment:
**I actually think we can do away with most of these, maybe leaving one as an example!
```suggestion
# Example: 
# AIRFLOW_VAR_API_KEY_BROOKLYN_MUSEUM=apikey
```